This fixes failures of tests "BaseLine_opt-datetime-format" and
"BaseLine_opt-time-report", which were printing an unexpected trailing '*' on
the last line of output, e.g.:

  @@ -5,4 +5,4 @@
     04/05/13 12:00 PM    04/05/13 01:30 PM       1.50h    Lunch
     04/05/13 11:30 AM    04/05/13 12:00 PM       30.0m    Walk
   --------------------------------------------------
  -                                           
  +                                          *

Reported upstream at
https://groups.google.com/d/msg/ledger-cli/EeJUrUk8YDc/pIR-LOTVEAAJ

diff --git a/src/account.h b/src/account.h
index 1b97463d..f2555593 100644
--- a/src/account.h
+++ b/src/account.h
@@ -187,7 +187,7 @@ public:
 
       datetime_t         earliest_checkin;
       datetime_t         latest_checkout;
-      bool               latest_checkout_cleared;
+      bool               latest_checkout_cleared = false;
 
       std::set<path>     filenames;
       std::set<string>   accounts_referenced;
